File: PythonFiles/Homework1.py
from math import sqrt
def TrabbPardoKnuth(S):
    for e in reversed(S):
        result = sqrt(abs(e)) + 5 * e
        if result > 500:
            print ("Error: incorrect sequence")
            break
        else:
            print (result)

def main():
    inputList = []
    print("Input 11 numbers:")
    for i in range(0, 11):
        val = ""
        while not val.isnumeric():
            val = input()
        inputList.append(int(val))
    TrabbPardoKnuth(inputList)

if __name__ == "__main__":
    main()

File: CSharp/Homework1.cs
using System;
using System.collections;

class Homework1
{
    public static void main(String args[]){
        
    }
    public static void TrabbPardoKnuth(List<int> sequence){
        for (int i = sequence.Count(); i > 0; i--){
            float result = Math.sqrt(abs(sequence[i])) + 5 * sequence[i];
            if (result > 500){
                throw new InvalidArgumentException("Sequence is incorrect");
            }else{
                Console.WriteLine(result);
            }
        }
    }
}